### PR TITLE
Add IsEmpty expander

### PR DESCRIPTION
**Pull request by blazarecki**
This PR add a `IsEmpty` expander.

Does I miss something ?

### CODE REVIEW

File: tests/Coduo/PHPMatcher/MatcherTest.php [hunk 1]
```diff
@@ -253,7 +253,9 @@ public static function expanderExamples()
             array(100, "@integer@.lowerThan(101).greaterThan(10)", true),
             array("", "@string@.notEmpty()", false),
             array("lorem ipsum", "@string@.notEmpty()", true),
+            array("", "@string@.isEmpty()", true),
             array(array("foo", "bar"), "@array@.inArray(\"bar\")", true),
+            array(array(), "@array@.isEmpty()", true),
```
Comment (norzechowicz) [inline tests/Coduo/PHPMatcher/MatcherTest.php]: could you please also add test that proves that for non empty valute this expander returns false?

Comment (blazarecki) [inline tests/Coduo/PHPMatcher/MatcherTest.php]: yes, it's done.
